sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations.py

```diff
@@ -160,6 +160,7 @@ def begin_create(
         self,
         workspace: Workspace,
         update_dependent_resources: bool = False,
+        byo_open_ai_resource_id: Optional[str] = None,
```

Comment:
Can you hide this in `kwargs` for now before we're ready to release this feature?